### ISSUE TITLE
/Screens?

### ISSUE THREAD

**Issue by ldmyers** [ibiggy9/Flourish-Mental-Health-Companion#1]
Am I missing the /Screens directory somewhere?

**Comment by ibiggy9** [comment]
Yes, thanks for the comment. I'm working to add the entire code in its entirety over time as there are still a number of missing files. The file structure is now fixed though.
